cli_tools_tests/e2e/gce_image_import_export/test_suites/export/image_export_test_suite.go

```diff
@@ -70,6 +72,14 @@ func TestSuite(ctx context.Context, tswg *sync.WaitGroup, testSuites chan *junit
 		testsMap[testType][imageExportWithSubnetWithoutNetworkTestCase] = runImageExportWithSubnetWithoutNetworkParamsTest
 	}
 
+	// Only test service account scenario for wrapper, until gcloud support it.
+	imageExportRawWithoutDefaultServiceAccountTestCase := junitxml.NewTestCase(
+		testSuiteName, fmt.Sprintf("[%v][ImageExport] %v", e2e.Wrapper, "Export Raw without default service account"))
+	imageExportVMDKWithoutDefaultServiceAccountPermissionTestCase := junitxml.NewTestCase(
+		testSuiteName, fmt.Sprintf("[%v][ImageExport] %v", e2e.Wrapper, "Export VMDK without default service account permission"))
```

Comment:
What does it mean "without default service account permission"? Default service account is used for the test but it doesn't have appropriate permissions?

Reply:
default service account is not deleted, but its permission (editor for example) is removed

Reply:
Maybe: imageExportVMDKDefaultServiceAccountWithMissingPermissionsTestCase would be more readable and avoid confusion if permissions or service account are missing?

Reply:
sounds good, changed accordingly